localstack/dev/run/__main__.py

```diff
@@ -234,6 +234,15 @@ def run(
         network=network,
     )
 
+    # replicate pro startup
+    if pro:
+        try:
+            from localstack_ext.plugins import modify_edge_port_config
+
+            modify_edge_port_config(config)
+        except ImportError:
+            pass
+
```

Comment:
Why is this necessary? Shouldn't this take place in the container then anyway?

Reply:
It is in a `configure_localstack_container` hook which is not run in this dev container configuration cc @thrau

Reply:
Ah, you are right, it is usually called as configure hook as well. thanks for the clarification!